### PR TITLE
bpo-25066: Added repr for multiprocessing.Event

### PR DESCRIPTION
**Pull request by kumaraditya303**
https://bugs.python.org/issue25066

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).



## CLA Missing

Our records indicate the following people have not signed the CLA:

@kumaraditya303

For legal reasons we need all the people listed to sign the CLA before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Lib/multiprocessing/synchronize.py [hunk 1]
```diff
@@ -353,6 +353,10 @@ def wait(self, timeout=None):
                 return True
             return False
 
+    def __repr__(self) -> str:
+        cls = self.__class__
+        set_status = 'set' if self.is_set() else 'unset'
+        return f"<{cls.__module__}.{cls.__name__} at {id(self):#x} {set_status}>"
```
Comment (tiran) [inline Lib/multiprocessing/synchronize.py]:
```suggestion
        set_status = 'set' if self.is_set() else 'unset'
        return f"<{type(self).__qualname__} at {id(self):#x} {set_status}>"
```

**Comment by kumaraditya303** [comment]
@pablogsal Can you review this?, Thanks

**Review by pablogsal — COMMENTED** [review]
LGTM

File: Misc/NEWS.d/next/Library/2021-11-24-12-25-42.bpo-25066.YIcIkn.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Added repr for ``multiprocessing.Event``, patch by Kumar Aditya.
```
Comment (pablogsal) [inline Misc/NEWS.d/next/Library/2021-11-24-12-25-42.bpo-25066.YIcIkn.rst]:
```suggestion
Added a :meth:`__repr__` method to  :class:`multiprocessing.Event` objects, patch by Kumar Aditya.
```

**Pull request merged**